docs/design/features/additional-deps.md

```diff
@@ -80,8 +80,8 @@ Where "found" means the version that is being used at run time including roll-fo
 ## 2.1 proposal (add an "any" tfm to store)
 For example,
     `\dotnet\store\x64\any\microsoft.applicationinsights\2.4.0`
-    
-The `any` tfm would be used if the specified tfm (e.g. netcoreapp2.0) is not found:    
+
+The `any` tfm would be used if the specified tfm (e.g. netcoreapp2.0) is not found:
     `\dotnet\store\x64\netcoreapp2.0\microsoft.applicationinsights\2.4.0`
```

Comment:
These was in separate lines, but now are in the same line after removing the trailing whitespaces.
Do you want them to be in separate lines again?

Reply:
Either way looks fine to me. It would be nice to make the rendered view and raw text look the same.